aiven/client/cli.py

```diff
@@ -1488,9 +1511,11 @@ def service__cli(self):
             command, params, env = self._build_influx_start_info(url)
         elif service_type == "postgres":
             command, params, env = self._build_psql_start_info(url)
+        elif service_type == "rediss":
```

Comment:
Maybe we should support `redis` as well? It seems some projects use `redis` instead of `rediss`.

Reply:
I don't think we're using bare `redis` (i.e. non TLS wrapped) connection in Aiven platform, so this should be fine